docs/resources/oms_migration_sync_task.md

```diff
@@ -0,0 +1,180 @@
+---
+subcategory: "Object Storage Migration Service (OMS)"
+---
+
+# huaweicloud_oms_migration_sync_task
+
+Manages an OMS migration synchronization task resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "source_region" {}
+variable "source_bucket" {}
+variable "source_access_key" {}
+variable "source_secret_key" {}
+variable "dest_region" {}
+variable "dest_bucket" {}
+variable "dest_access_key" {}
+variable "dest_secret_key" {}
+
+resource "huaweicloud_oms_migration_sync_task" "test" {
+  src_cloud_type = "HuaweiCloud"
+  region         = var.dest_region
+  src_ak         = var.source_access_key
+  src_sk         = var.source_secret_key
+  src_region     = var.source_region
+  src_bucket     = var.source_bucket
+  dst_ak         = var.dest_access_key
+  dst_sk         = var.source_secret_key
+  dst_bucket     = var.dest_bucket
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource. Which is also the region
+  for the destination bucket. If omitted, the provider-level region will be used.
+  Changing this creates a new resource.
+
+* `src_region` - (Required, String, ForceNew) Specifies the region where the source bucket is located.
+  Changing this creates a new resource.
+
+* `src_bucket` - (Required, String, ForceNew) Specifies the name of the source bucket.
+  Changing this creates a new resource.
+
+* `src_ak` - (Required, String, ForceNew) Specifies the access key for accessing the source bucket.
+  Changing this creates a new resource.
+
+* `src_sk` - (Required, String, ForceNew) Specifies the secret key for accessing the source bucket.
+  Changing this creates a new resource.
+
+* `dst_bucket` - (Required, String, ForceNew) Specifies the name of the destination bucket.
+  Changing this creates a new resource.
+
+* `dst_ak` - (Required, String, ForceNew) Specifies the access key for accessing the destination bucket.
+  Changing this creates a new resource.
+
+* `dst_sk` - (Required, String, ForceNew) Specifies the secret key for accessing the destination bucket.
+  Changing this creates a new resource.
+
+* `app_id` - (Optional, String, ForceNew) Specifies the APP ID. This parameter is mandatory when `src_cloud_type` is
```

Comment:
it's better to add `src_cloud_type` before `app_id`.

Reply:
done